components/builder-protocol/protocols/scheduler.proto

```diff
@@ -0,0 +1,20 @@
+package scheduler;
+import "protocols/depotsrv.proto";
+
+message Schedule {
+  required depotsrv.PackageIdent ident = 1;
```

Comment:
Makes sense.  Google seems to have followed the recommendation of the faction of their engineers who believe required was more trouble than it was worth, when long term maintenance costs were taken into account and removed it entirely. So, optional looks like a good way to go for our message fields.